attributes/default.rb

```diff
@@ -10,30 +10,33 @@
   default["sensu"]["windows"]["package_options"] = nil
   default["sensu"]["windows"]["install_dotnet"] = true
   default["sensu"]["windows"]["dotnet_major_version"] = 4
+  default["sensu"]["version"] = "1.0.2-1-x64"
```

Comment:
Is this required to use the new logrotate? I'd prefer to keep the changes as isolated as possible. I am not sure that we should default to 1.x yet as there are a handful of community plugins that have conflicts and need to be worked out first. You can always overwrite the version in your wrapper cookbook.

Reply:
Logrotate is supported as of 0.29.5 (https://github.com/sensu/sensu-chef/issues/580). I checked the repo url and it has been changed. If I look at http://repositories.sensuapp.org/msi/. I see 1 type of msi for both 32/64 bit, now there are 2 msi's (http://repositories.sensuapp.org/msi/2016/). I had to test this with 1.0.2 as we have had issues with pre 1.x versio and already installed 1.0.2, which was latest at that moment of time. Reverting back to 0.28.4-1 would mean that logrotate won't work.

Reply:
> I see 1 type of msi for both 32/64 bit

I believe these are 32bit builds which work on 64bit systems. We now build both 32 and 64 bit installers.